src/components/fields/select-field/README.md

```diff
@@ -0,0 +1,83 @@
+# SelectField
+
+#### Description
+
+A controlled input component with validation states and a label getting a selection from the user.
```

Comment:
maybe:
> A controlled input component with validation states and a label getting a `select value` from the user.

Reply:
This is based on `SelectInput` where it is already that way.

Reply:
no worries. was just a minor suggestion. looks good enough